src/parser/jobs/mnk/modules/CelestialRevolution.tsx

```diff
@@ -0,0 +1,42 @@
+import {Plural, Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+export class CelestialRevolution extends Analyser {
+	static override handle = 'celestialRevolution'
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	private count: number = 0
```

Comment:
Not a fan of generic names, prefer it to be explicit.
```suggestion
	private revolutions: number = 0
```